### PR TITLE
IsSwipeNavigationEnabled spec draft

### PR DESCRIPTION
**Pull request by tofuandeve**
This is a draft spec for Swipe Navigation API

### CODE REVIEW

File: specs/IsSwipeNavigationEnabled.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  /// 2. Pull to refresh (swipe vertically) the current page. (This feature is
+  /// currently disabled by default in the browser, to enable in WebView2, use
+  /// `put_AdditionalBrowserArguments` api with "--pull-to-refresh" switch).
+  /// The `IsSwipeNavigationEnabled` property enables or disables the ability of the
+  /// end user to use swiping gesture on touch input enabled devices to
+  /// navigate in WebView2. It defaults to `TRUE`.
+  /// When set to `FALSE`, the end user cannot swipe to navigate or pull to refresh.
+  /// This API only affects the overscrolling navigation functionality and has no
+  /// effect on the scrolling interaction used to explore the web content shown
+  /// in WebView2.
```
Comment (david-risney) [inline specs/IsSwipeNavigationEnabled.md:71]: Does modification of this setting only apply after the next page navigation?

File: specs/IsSwipeNavigationEnabled.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  ///
```
Comment (david-risney) [inline specs/IsSwipeNavigationEnabled.md:72]: Does modification of this setting only apply to the current webview or all webviews in the environment? If just this webview, then no need to specify anything. If it applies to all webviews in the environment then need to say that in the documentation.

**Review by david-risney — CHANGES_REQUESTED** [review]
Generally looks good. Just two questions to answer and then complete this PR and open the next PR to main and let me know. Thx

File: specs/IsSwipeNavigationEnabled.md (hunk 2 above)
Comment (tofuandeve) [reply specs/IsSwipeNavigationEnabled.md:72]: This setting change only applies to the current webview 😀

Comment (david-risney) [reply specs/IsSwipeNavigationEnabled.md:75]: 👍👍

**Review by david-risney — APPROVED** [review]

**Pull request merged**